docs/arc42/12_glossary.md

```diff
@@ -4,5 +4,38 @@
 |----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | AA       | amino acid                                                                                                                                                                                                                                                       |
 | aligned  | a nucleotide sequence is aligned, if it is arranged such that it has many similarities to a given reference genome. The aligned sequence has the same length as the reference genome. Gaps are marked in the aligned sequence. Insertions are stored separately. |
+| mutation | a divergence from the reference genome (see below).                                                                                                                                                                                                              |
 | QC       | quality control; in our case, it usually refers to the quality checks and metrics of the sequences, targeting how well the nucleotide sequence was determined from the probe.                                                                                    |
 | Variant  | We follow a very open definition of variants. Every subset of sequences is considered a variant. A variant is specified by lineage/clade names and mutations. A variant does not need to be [monophyletic](https://en.wikipedia.org/wiki/Monophyly).             |
+
+## Mutations
+
+Mutations can occur either on the nucleotide level or on the AA level. For the nucleotides a single symbol can produce a
+mutation, whereas for the amino acids, some nucleotide mutations still produce the same amino
+acid ([see also](https://en.wikipedia.org/wiki/DNA_and_RNA_codon_tables)).
+
+The following explains the notations for mutations.
+
+**Amino acid**
+
+The gene has to be provided for the AA mutation, since AAs only make sense within a gene.
+
+Example ORF_1a:G1234S translates to
+
+- in Gene: ORF_1a
+- AA mutation from G to S at position 1234
```

Comment:
Does the position actually start at 0 or at 1?

Reply:
It starts at 1

Reply:
I think SILO currently starts at 0:

```
POST http://localhost:8081/query
Content-Type: application/json

{
  "action": {
    "type": "Mutations"
  },
  "filterExpression": {
    "type": "True"
  }
}
```
returns
```
{
  ...,
  "queryResult": [
    {
      "count": 45,
      "mutation": "A0-",
      "proportion": 0.9782608695652174
    },
    {
      "count": 44,
      "mutation": "T1-",
      "proportion": 0.9565217391304348
    },
...
]}
```
@Taepper we should check that.

Reply:
Good point. The filter should consider this (although the logic for that should probably be moved from the `getBitmap`-call to the parsing phase).
For the mutation distribution this should be definitely be changed. But the change should also happen when building the resulting string of {from_symbol}{position}{to_symbol}, not when returning the mutation struct.